### ISSUE TITLE
Superscript/subscript immediately following italics/bold text is incorrectly italics/bold

### ISSUE THREAD

**Issue by sevenshamrocks** [evbacher/gd2md-html#152]
When writing formatted text immediately followed by unformatted superscript/subscript text, the superscript/subscript incorrectly has the formatting of the preceding text. 

Ex: 

**BOLD**<sup>superscript</sup>

becomes

**BOLD<sup>superscript</sup>**

*ITALICS*<sup>superscript</sup>

becomes

*ITALICS<sup>superscript</sup>*

**Comment by evbacher** [comment]
Yes, I can see this -- thanks for reporting. I am making some larger changes to Docs to Markdown, but I'll try to get to this bug soon.

**Comment by sevenshamrocks** [comment]
Thanks. I appreciate the development time. I forked the project and I'm going to see if I can figure out some other tweaks/issues I've had.

**Comment by evbacher** [comment]
Fixed (https://github.com/evbacher/gd2md-html/pull/166)
